db scripts/Contact_functions/update_contact_person.sql

```diff
@@ -9,11 +9,11 @@ firstName varchar;
 lastName varchar;
```

Comment:
Daniel Landau updates this function in his pr, maybe he should upload this stuff

Reply:
He cant because this change isn't on dev code yet because it wasnt merged...